### PR TITLE
[bug] Harmonize dialog boxes across the application.

### PR DESCRIPTION
**Pull request by chibujax**
# JIRA TICKET NAME:
PDMO-30: [Harmonize dialog boxes across the application](https://issues.openmrs.org/browse/PDMO-30)

## SUMMARY:
Harmonize dialog boxes across the application

### CODE REVIEW

File: app/js/components/obs.jsx [hunk 1]
```diff
@@ -446,29 +337,41 @@ export default class Obs extends React.Component {
...
               </div>
-            </ModalDialog>
-          </ModalContainer>
+              <div className="modal-footer">
+                <div className="col-sm-6">
+                  <button
+                    type="button" className="btn btn-success form-control"
+                    onClick={this.delete}
+                    data-dismiss="modal"
+                  >Confirm
```
Comment (jacksono) [inline app/js/components/obs.jsx]: Indentation to be fixed here

File: app/js/components/obs.jsx [hunk 2]
```diff
@@ -446,29 +337,41 @@ export default class Obs extends React.Component {
...
+                  </button>
+                </div>
+                <div className="col-sm-6">
+                  <button
+                    type="button" className="btn btn-danger form-control cancelBtn"
+                    data-dismiss="modal"
+                  >Close
```
Comment (jacksono) [inline app/js/components/obs.jsx]: Looks like a tab is missing here

File: app/js/components/subAddress/addressForm.jsx [hunk 3]
```diff
@@ -169,20 +150,38 @@ export default class AddressForm extends React.Component {
...
+                  <p> Are you sure want to delete this address?</p>
+                </div>
+                <div className="modal-footer">
+                  <div className="col-sm-6">
+                    <button
+                      type="button"
+                      className="btn btn-success form-control"
+                      onClick={this.delete}
+                      data-dismiss="modal"
+                    >Confirm
```
Comment (jacksono) [inline app/js/components/subAddress/addressForm.jsx]: Indentation fix needed here

File: app/js/components/subAddress/addressForm.jsx [hunk 4]
```diff
@@ -169,20 +150,38 @@ export default class AddressForm extends React.Component {
...
+                    </button>
+                  </div>
+                  <div className="col-sm-6">
+                    <button
+                      type="button"
+                      className="btn btn-danger form-control cancelBtn"
+                      data-dismiss="modal"
+                    >Close
```
Comment (jacksono) [inline app/js/components/subAddress/addressForm.jsx]: Ditto

**Comment by jacksono** [comment]
Generally looks good, good job.

**Review by jacksono — CHANGES_REQUESTED** [review]

File: app/js/components/obs.jsx (hunk 2 above)
Comment (chibujax) [reply app/js/components/obs.jsx]: Kindly view the file, the indentation is two white space as specified here https://github.com/felixge/node-style-guide#2-spaces-for-indentation referenced https://wiki.openmrs.org/display/docs/JavaScript+Conventions

File: app/js/components/subAddress/addressForm.jsx (hunk 3 above)
Comment (chibujax) [reply app/js/components/subAddress/addressForm.jsx]: Indentation rule was used https://github.com/felixge/node-style-guide#2-spaces-for-indentation referenced https://wiki.openmrs.org/display/docs/JavaScript+Conventions

File: app/js/components/subAddress/addressForm.jsx (hunk 4 above)
Comment (chibujax) [reply app/js/components/subAddress/addressForm.jsx]: Same as above, thank you

**Pull request closed without merging**